package.json

```diff
@@ -48,7 +48,7 @@
 		"@godaddy/terminus": "^4.11.0",
 		"@graphql-tools/load": "^7.7.0",
 		"@graphql-tools/url-loader": "^7.12.1",
-		"@kiva/kv-components": "^3.1.0",
+		"@kiva/kv-components": "^3.2.0",
```

Comment:
@christian14b I think you may need to run `npm i` so that this updates the package-lock file too.

Reply:
The release happened and this version is on npm https://www.npmjs.com/package/@kiva/kv-components

Reply:
I think/hope this will help github action pass...